package/scripts/params.py

```diff
@@ -119,6 +119,15 @@
 
 router_hosts = config['clusterHostInfo']['cdap_router_hosts']
 router_hosts.sort()
-cdap_router_host = router_hosts[0]
+if len(router_hosts) > 1:
+    for i, val in enumerate(router_hosts):
```

Comment:
I can. I pretty much copied the code from above where we do the same thing for zookeeper/kafka... I had originally considered some method of trying to determine which router to use based on which host we are, to provide load balancing in the service, automatically, but decided against that added complexity.